src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/LinkWidget.tsx

```diff
@@ -1,29 +1,70 @@
+import _ from 'lodash'
 import React from 'react'
-import { AbstractLinkFactory, DefaultLinkModel, DefaultLinkWidget, PointModel } from 'storm-react-diagrams'
+import { AbstractLinkFactory, DefaultLinkModel, DefaultLinkWidget, PointModel, Toolkit } from 'storm-react-diagrams'
+
+import style from './style.scss'
 
 class DeletableLinkWidget extends DefaultLinkWidget {
-  addPointToLink = (event, index: number): void => {
-    if (
-      event.type === 'contextmenu' &&
-      !this.props.diagramEngine.isModelLocked(this.props.link) &&
-      this.props.link.points.length - 1 <= this.props.diagramEngine.getMaxNumberPointsPerLink()
-    ) {
-      event.preventDefault()
-      const point = new PointModel(this.props.link, this.props.diagramEngine.getRelativeMousePoint(event))
-      point.setSelected(true)
-      this.forceUpdate()
-      this.props.link.addPoint(point, index)
-      this.props.pointAdded(point, event)
-    }
+  private currentLink: string
+
+  generatePoint(pointIndex: number): JSX.Element {
+    const { link } = this.props
+    const x = link.points[pointIndex].x
+    const y = link.points[pointIndex].y
+
+    this.currentLink = link.getID()
+
+    return (
```

Comment:
what's that ?

Reply:
The fancy link between nodes, I just copy-pasted it from dev to have the link highlight for the debugger